Drop down issue

Jeelislive opened the pull request:
I solved drop issue please verify it once from your side.

mi5t4n commented:
Hello @Jeelislive,

Thank you for submitting the pull request. Please include recordings demonstrating the issue, as well as recordings showing the behavior after it has been resolved. This will be very helpful for our review process.

Jeelislive commented:
Hi @mi5t4n, thanks for your previous feedback on my earlier PR. I have addressed the issues and created this new PR. Please review when you get a chance. Also, I’ve included video recordings as you requested.
